pkg/sfu/buffer/rtpstats.go

```diff
@@ -1896,18 +1896,20 @@ func (p *PIDController) Update(setpoint, measurement float64, at time.Time) floa
 	p.prevError = errorTerm
 	p.prevMeasurement = measurement
 	p.prevMeasurementTime = at
-	p.logger.Debugw(
-		"pid controller",
-		"setpoint", setpoint,
-		"measurement", measurement,
-		"errorTerm", errorTerm,
-		"proportional", proportional,
-		"integral", iVal,
-		"integralLimited", boundIVal,
-		"derivative", p.dVal,
-		"output", output,
-		"outputLimited", boundOutput,
-	)
+	/*
+		p.logger.Debugw(
+			"pid controller",
+			"setpoint", setpoint,
+			"measurement", measurement,
+			"errorTerm", errorTerm,
+			"proportional", proportional,
+			"integral", iVal,
+			"integralLimited", boundIVal,
+			"derivative", p.dVal,
+			"output", output,
+			"outputLimited", boundOutput,
+		)
+	*/
```

Comment:
sorry, breaking my own PR comment rule there of just commenting instead of deleting. Leaving it to have a quick option to re-enable as it is logging a bunch of parameters and looking back at an older version is time consuming to restore.